tests/auth_tests/test_views.py

```diff
@@ -1440,14 +1440,35 @@ def test_user_change_password(self):
         response = self.client.get(user_change_url)
         # Test the link inside password field help_text.
         rel_link = re.search(
-            r'you can change the password using <a href="([^"]*)">this form</a>',
+            r'you can change or unset the password via <a href="([^"]*)">this form</a>',
             response.content.decode(),
         )[1]
         self.assertEqual(urljoin(user_change_url, rel_link), password_change_url)
 
+        response = self.client.get(password_change_url)
+        # Test the form title with original (usable) password
+        self.assertContains(response, "<h1>Change password")  # Title
+        self.assertContains(response, "&rsaquo; Change password")  # Breadcrumb
+        self.assertNotContains(response, "<h1>Set password")
+        self.assertNotContains(response, "&rsaquo; Set password")
+
+        # Test for title with unusable password with a test user
```

Comment:
Could we split this in two tests? one starting with a user with a usable password, and the other for a user without.

Reply:
Done.